nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/trace/TracerFactory.java

```diff
@@ -54,7 +55,7 @@
 
     protected String printable;
 
-    protected Function<String,Boolean> printableAssertor;
+    protected Function<String, Boolean> printableAssertor;
```

Comment:
Please do reformattings in separate commits in the future. Having formatting mixed with actual code changes makes it harder to review pull requests.